ebml.xml

```diff
@@ -0,0 +1,61 @@
+	<element name="DocType" path="\EBML\DocType" id="0x4282" type="string" default="ebml" minOccurs="1" maxOccurs="1">
+		<documentation lang="en" purpose="definition">A string that describes the type of document that follows this EBML header. 'matroska' in our case or 'webm' for webm files.</documentation>
+	</element>
+	<element name="DocTypeVersion" path="\EBML\DocTypeVersion" id="0x4287" type="uinteger" range="1-8" default="1" minOccurs="1" maxOccurs="1">
+		<documentation lang="en" purpose="definition">The version of DocType interpreter used to create the file.</documentation>
+	</element>
+	<element name="DocTypeReadVersion" path="\EBML\DocTypeReadVersion" id="0x4285" type="uinteger" range="1-8" default="1" minOccurs="1" maxOccurs="1">
+		<documentation lang="en" purpose="definition">The minimum DocType version an interpreter has to support to read this file.</documentation>
+	</element>
+
+	<element name="SignatureSlot" path="\EBML\SignatureSlot" id="0x1B538667" type="master">
+		<documentation lang="en" purpose="definition">Contain signature of some (coming) elements in the stream.</documentation>
+	</element>
+	<element name="SignatureAlgo" path="\EBML\SignatureSlot\SignatureAlgo" id="0x7E8A" type="uinteger">
+		<documentation lang="en" purpose="definition">Signature algorithm used (1=RSA, 2=elliptic).</documentation>
+	</element>
+	<element name="SignatureHash" path="\EBML\SignatureSlot\SignatureHash" id="0x7E9A" type="uinteger">
+		<documentation lang="en" purpose="definition">Hash algorithm used (1=SHA1-160, 2=MD5).</documentation>
+	</element>
+	<element name="SignaturePublicKey" path="\EBML\SignatureSlot\SignaturePublicKey" id="0x7EA5" type="binary">
+		<documentation lang="en" purpose="definition">The public key to use with the algorithm (in the case of a PKI-based signature).</documentation>
+	</element>
+	<element name="Signature" path="\EBML\SignatureSlot\Signature" id="0x7EB5" type="binary">
+		<documentation lang="en" purpose="definition">The signature of the data (until a new.</documentation>
+	</element>
+
+	<element name="SignatureElements" path="\EBML\SignatureElements" id="0x7E5B" type="master">
+		<documentation lang="en" purpose="definition">Contains elements that will be used to compute the signature.</documentation>
+	</element>
+	<element name="SignatureElementList" path="\EBML\SignatureElements\SignatureElementList" id="0x7E7B" type="master">
+		<documentation lang="en" purpose="definition">
+			A list consists of a number of consecutive elements that represent one case where data is used in signature. Ex: Cluster|Block|BlockAdditional means that the BlockAdditional of all Blocks in all Clusters is used for encryption.
+		</documentation>
+	</element>
+	<element name="SignedElement" path="\EBML\SignatureElements\SignatureElementList\SignedElement" id="0x6532" type="binary" >
+		<documentation lang="en" purpose="definition">An element ID whose data will be used to compute the signature.</documentation>
+	</element>
+
```

Comment:
It's missing `DocTypeExtension`, `DocTypeExtensionName`, `DocTypeExtensionVersion`.